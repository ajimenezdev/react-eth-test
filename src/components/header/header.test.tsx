import React from "react";
import renderer from "react-test-renderer";
import Header from "./index";

it("renders without crashing", () => {
  const component = renderer.create(<Header />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
