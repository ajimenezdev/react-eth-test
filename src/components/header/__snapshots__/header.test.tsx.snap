// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<header
  className="App-header"
>
  <p>
    React-Ethereum test
  </p>
</header>
`;
